### DISCUSSION TITLE
Thunderbird does not delete messages on the server

### DISCUSSION THREAD

**Discussion by lexesv** [emersion/maddy#435] [Q&A]
Hello.

I set in Thunderbird Config editor: **mail.imap.expunge_after_delete;true**

But messages are still not deleted from the server.

**Comment by cbrake** [comment]
Thunderbird seems to be deleting messages on server fine for me. What TB version are you running?

I'm running 91.6.1 on Arch linux

**Reply by lexesv** [reply]
91.5.0 (64-bit) on Debian 11

**Reply by cbrake** [reply]
Very odd. 

Another note -- I'm just using the stock conf file that comes with maddy.

I did see one case with my old mail server where one TB instance would delete files in one folder, but another would not.

Another test might be to connect from a fresh TB instance and see if it does the same things -- could be some obscure TB config.
